app/public/lobby/index.html

```diff
@@ -3,15 +3,56 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" type="text/css" href="style.css">
     <title>Lobby</title>
 </head>
 <body>
-    One sec, deciding your role
+	<div id="home">
+		<header>
+			<div class="header-cont">
+				<h2 class="title header-elem">Cyberspace Saboteur</h2>
+				<div class="header-info header-elem">
+					<h2 class="name header-elem">username here?</h2>
+					<button class="header-btn header-elem">?</button>
+					<button class="header-btn header-elem">?</button>
+					<button class="header-btn header-elem">?</button> <!--possible header btns for settings, stats, idk-->
+				</div>
+			</div>
+		</header>
+		<div>
+			<h2 class="lobby-header">Lobby</h2>
+		</div>
+		<div>
+			<div class="container">
+				<div class="chat page-cont"></div>
```

Comment:
I like the idea of having a chat panel like you showed in the Set game. But that might make us responsible for moderating/censoring it😳
Does Set have chat moderation?